views/signedup.tmpl

```diff
@@ -0,0 +1,100 @@
+{{define "title"}}{{.PageContext.ShortName}} | Sign Up{{end}}
+
+{{define "open-graph"}}
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@{{.PageContext.MainTwitter}}">
+<meta name="twitter:creator" content="@{{.PageContext.MainTwitter}}">
+<meta property="og:type" content="website">
+<meta property="og:locale" content="en_GB">
+<meta property="og:title" content="{{.PageContext.LongName}} | Sign Up">
+<meta property="og:description" content="{{.PageContext.SiteDescription}}">
+<meta property="og:image" content="{{.PageContext.FullURL}}{{url "/images/open_graph-general.jpg"}}">
+<meta property="og:url" content="{{.PageContext.FullURL}}{{url "/signup/"}}">
```

Comment:
Not sure the OG stuff is needed here as it looks the same as whats in the base template except for the sign up and as this is the "signed-up" page it should really be a thing that's shared.

Reply:
We should look at using the title block in the OG data of the base template so we don't have to duplicate the OG data just to set the title